stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -118,12 +247,41 @@ select = (column="_value", fn=(column, tables=<-) => tables, as, tables=<-) => {
         |> rename(fn: (column) => if column == _column then _as else column)
 }
 
-// selectWindow selects a column with time grouping and computes aggregated values.
-// It is a convenience function to be used as
+// selectWindow changes a column’s name, windows rows by time,
+// and applies an aggregate or selector function the specified column for each window of time.
+//
+// ## TICKscript helper function
+// `tickscript.selectWindow` is a helper function meant to replicate TICKscript operations like the following:
+// ```
+// Rename, window, and aggregate
+// query("SELECT f(x) AS y")
+//   .groupBy(time(t), ...)
+// ```
```

Comment:
```suggestion
// selectWindow changes a column’s name, windows rows by time, and then applies an
// aggregate or selector function the specified column for each window of time.
//
// ## TICKscript helper function
// `tickscript.selectWindow` is a helper function meant to replicate TICKscript operations like the following:
//
// ```
// // Rename, window, and aggregate
// query("SELECT f(x) AS y")
//   .groupBy(time(t), ...)
// ```
```